### PR TITLE
48034_show_subject_activity_by_public_activity

### PR DESCRIPTION
**Pull request by zenith19**

### CODE REVIEW

**Comment by Rownaknira** [comment]
ok

File: app/models/subject.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class Subject < ActiveRecord::Base
+  
+  include PublicActivity::Model
+  tracked owner: ->(controller, model) { controller && controller.current_user }
```
Comment (MuhammadTamzid) [inline app/models/subject.rb]: Convention
{controller && controller.current_user}

File: app/models/subject.rb [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+  tracked recipient: ->(controller, model) { model && model }
```
Comment (MuhammadTamzid) [inline app/models/subject.rb]: Convention
{model && model}

File: app/views/supervisor/subjects/_subjects_activities.html.erb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<div class="panel panel-default">
+  <div class="panel-body">
+  <h2><%= "Activities" %></h2>
```
Comment (MuhammadTamzid) [inline app/views/supervisor/subjects/_subjects_activities.html.erb]: ??

**Pull request merged**